newIDE/app/src/AssetStore/AssetsList.js

```diff
@@ -140,14 +143,25 @@ const styles = {
     display: 'flex',
     flexDirection: 'column',
   },
+  previewImageContainer: {
+    display: 'flex',
+    flex: 0.7,
```

Comment:
Empirical value I guess? 😅

Reply:
Yeah it's to have the image take less than half the space, but still something